app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -482,28 +483,43 @@ public class MainActivity : AppCompatActivity(), ViewController, Router.Callback
         startActivityForResult(intent, requestCodeSearchResults)
     }
 
-    override fun showRoutingMode() {
+    override fun showRoutingMode(feature: Feature) {
```

Comment:
makes sense - it'll also encourage use of the presenter rather than keeping control flow in the mainactivity